config/initializers/gitlab_shell_secret_token.rb

```diff
@@ -14,6 +13,8 @@
   File.write(secret_file, token)
 end
 
-if File.exist?(Gitlab.config.gitlab_shell.path) && !File.exist?(gitlab_shell_symlink)
-  FileUtils.symlink(secret_file, gitlab_shell_symlink)
+# TODO May be removed after updating the default secret_file path...?
```

Comment:
To be honest, I still can’t understand why the heck someone has chosen to create symlink instead of simply using the same path…